### ISSUE TITLE
Problem with anvi-get-sequences-for-hmm-hits: Library not loaded: @rpath/libopenblas.dylib

### ISSUE THREAD

**Issue by scottdaniel** [merenlab/anvio#1200]
Hi guys,

I'm on Mac OS Mojave 10.14.5 and have installed anvio using conda. I'm getting an error when running through the tutorial here http://merenlab.org/2017/06/07/phylogenomics/

Specifically, I get the error when using the command `anvi-get-sequences-for-hmm-hits` full output below...

```
Anvi'o version ...............................: margaret (v5.5)
Profile DB version ...........................: 31
Contigs DB version ...........................: 12
Pan DB version ...............................: 13
Genome data storage version ..................: 6
Auxiliary data storage version ...............: 2
Structure DB version .........................: 1

(anvio) 54ee759272b2:distantly-related danielsg$ anvi-get-sequences-for-hmm-hits --external-genomes external-genomes.txt                                    --list-hmm-sources
Traceback (most recent call last):
  File "/Users/danielsg/miniconda3/envs/anvio/bin/anvi-get-sequences-for-hmm-hits", line 47, in <module>
    import anvio.hmmopswrapper as hmmopswrapper
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/hmmopswrapper.py", line 13, in <module>
    from anvio.genomedescriptions import GenomeDescriptions
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/genomedescriptions.py", line 22, in <module>
    import anvio.summarizer as summarizer
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/summarizer.py", line 50, in <module>
    import anvio.completeness as completeness
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/completeness.py", line 18, in <module>
    import anvio.scgdomainclassifier as scgdomainclassifier
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/scgdomainclassifier.py", line 20, in <module>
    from anvio.learning import RF
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/anvio/learning.py", line 16, in <module>
    import sklearn.ensemble
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/ensemble/__init__.py", line 7, in <module>
    from .forest import RandomForestClassifier
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/ensemble/forest.py", line 57, in <module>
    from ..metrics import r2_score
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/__init__.py", line 33, in <module>
    from . import cluster
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/cluster/__init__.py", line 20, in <module>
    from .unsupervised import silhouette_samples
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/cluster/unsupervised.py", line 12, in <module>
    from ..pairwise import pairwise_distances
  File "/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/pairwise.py", line 30, in <module>
    from .pairwise_fast import _chi2_kernel_fast, _sparse_manhattan
ImportError: dlopen(/Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/pairwise_fast.cpython-36m-darwin.so, 2): Library not loaded: @rpath/libopenblas.dylib
  Referenced from: /Users/danielsg/miniconda3/envs/anvio/lib/python3.6/site-packages/sklearn/metrics/pairwise_fast.cpython-36m-darwin.so
  Reason: image not found
```
 
Here are the details of my conda anvio environment:

```
packages in environment at /Users/danielsg/miniconda3/envs/anvio:
# Name                    Version                   Build  Channel
anvio                     5.5.0                         0    bioconda
anvio-minimal             5.5.0            py36h94a39f0_0    bioconda
appdirs                   1.4.3                      py_1    conda-forge
asn1crypto                0.24.0                py36_1003    conda-forge
attrs                     19.1.0                     py_0    conda-forge
bcftools                  1.9                  h3a161e8_4    bioconda
biopython                 1.74             py36h01d97ff_0    conda-forge
blas                      2.10                   openblas    conda-forge
blast                     2.9.0           pl526hae12ce6_3    bioconda
boost                     1.70.0           py36haf112f3_1    conda-forge
boost-cpp                 1.70.0               h75728bb_2    conda-forge
bottle                    0.12.13                    py_1    conda-forge
bowtie2                   2.3.5            py36h5c9b4e4_0    bioconda
bwa                       0.7.17               ha441bb4_5    bioconda
bzip2                     1.0.8                h01d97ff_0    conda-forge
ca-certificates           2019.6.16            hecc5488_0    conda-forge
centrifuge                1.0.4_beta      py36pl526ha92aebf_2    bioconda
certifi                   2019.6.16                py36_1    conda-forge
cffi                      1.12.3           py36hccf1714_0    conda-forge
chardet                   3.0.4                 py36_1003    conda-forge
cherrypy                  8.0.0                    py36_0    conda-forge
colored                   1.3.93                     py_0    conda-forge
configargparse            0.13.0                     py_1    conda-forge
coreutils                 8.25                          1    bioconda
cryptography              2.7              py36h212c5bf_0    conda-forge
curl                      7.65.3               h22ea746_0    conda-forge
cycler                    0.10.0                     py_1    conda-forge
cython                    0.29.13          py36h6de7cb9_0    conda-forge
datrie                    0.7.1            py36h1de35cc_0    conda-forge
dbus                      1.13.6               h2f22bb5_0    conda-forge
diamond                   0.9.25               hf79dc1e_0    bioconda
django                    2.0.8                    py36_0    conda-forge
docutils                  0.15.1                   py36_0    conda-forge
entrez-direct             11.0                    pl526_0    bioconda
ete3                      3.1.1                    py36_0    bioconda
expat                     2.2.5             h6de7cb9_1003    conda-forge
freetype                  2.10.0               h24853df_0    conda-forge
gettext                   0.19.8.1          h46ab8bc_1002    conda-forge
glib                      2.58.3            h9d45998_1002    conda-forge
gsl                       2.4               ha2d443c_1006    conda-forge
h5py                      2.9.0           nompi_py36h35dc34b_1102    conda-forge
hdf5                      1.10.4          nompi_h0cbb7df_1106    conda-forge
hmmer                     3.2.1                h0a44026_1    bioconda
htslib                    1.9                  h3a161e8_7    bioconda
icu                       64.2                 h6de7cb9_0    conda-forge
idna                      2.7                   py36_1002    conda-forge
illumina-utils            2.6                        py_0    bioconda
iqtree                    1.6.11               h5c9b4e4_0    bioconda
jpeg                      9c                h1de35cc_1001    conda-forge
jsonschema                3.0.1                    py36_0    conda-forge
kiwisolver                1.1.0            py36h770b8ee_0    conda-forge
krb5                      1.16.3            hcfa6398_1001    conda-forge
libblas                   3.8.0               10_openblas    conda-forge
libcblas                  3.8.0               10_openblas    conda-forge
libcurl                   7.65.3               h16faf7d_0    conda-forge
libcxx                    8.0.0                         4    conda-forge
libcxxabi                 8.0.0                         4    conda-forge
libdeflate                1.0                  h1de35cc_1    bioconda
libedit                   3.1.20170329      hcfe32e1_1001    conda-forge
libffi                    3.2.1                         1    bioconda
libgfortran               3.0.1                         0    conda-forge
libiconv                  1.15              h01d97ff_1005    conda-forge
libidn2                   2.1.1                h1de35cc_0    conda-forge
liblapack                 3.8.0               10_openblas    conda-forge
liblapacke                3.8.0               10_openblas    conda-forge
libopenblas               0.3.6                hd44dcd8_5    conda-forge
libpng                    1.6.37               h2573ce8_0    conda-forge
libssh2                   1.8.2                hcdc9a53_2    conda-forge
libunistring              0.9.10               h1de35cc_0    conda-forge
libxcb                    1.13              h1de35cc_1002    conda-forge
libxml2                   2.9.9                h12c6b28_2    conda-forge
libxslt                   1.1.32            h320ff13_1004    conda-forge
llvm-openmp               8.0.0                h770b8ee_0    conda-forge
lxml                      4.3.4            py36h08abf6f_0    conda-forge
matplotlib                2.2.3                    py36_1    conda-forge
matplotlib-base           2.2.3            py36hd64abcd_1    conda-forge
mcl                       14.137          pl526h470a237_4    bioconda
megahit                   1.2.6                hfbae3c0_0    bioconda
mistune                   0.8.1            py36h638d0ca_0  
mummer                    3.23                          8    bioconda
muscle                    3.8.1551             h04f5b5a_4    bioconda
ncurses                   6.1               h0a44026_1002    conda-forge
nose                      1.3.7                 py36_1002    conda-forge
numpy                     1.16.4           py36h6b0580a_0    conda-forge
openblas                  0.3.6                hd44dcd8_5    conda-forge
openmp                    8.0.0                         0    conda-forge
openssl                   1.1.1c               h01d97ff_0    conda-forge
pandas                    0.25.0           py36h86efe34_0    conda-forge
patsy                     0.5.1                      py_0    conda-forge
pcre                      8.41              h0a44026_1003    conda-forge
perl                      5.26.2            haec8ef5_1006    conda-forge
perl-app-cpanminus        1.7044                  pl526_1    bioconda
perl-archive-tar          2.32                    pl526_0    bioconda
perl-base                 2.23                    pl526_1    bioconda
perl-business-isbn        3.004                   pl526_0    bioconda
perl-business-isbn-data   20140910.003            pl526_0    bioconda
perl-carp                 1.38                    pl526_3    bioconda
perl-common-sense         3.74                    pl526_2    bioconda
perl-compress-raw-bzip2   2.086           pl526h0a44026_0    bioconda
perl-compress-raw-zlib    2.086           pl526h04f5b5a_1    bioconda
perl-constant             1.33                    pl526_1    bioconda
perl-data-dumper          2.173                   pl526_0    bioconda
perl-digest-hmac          1.03                    pl526_3    bioconda
perl-digest-md5           2.55                    pl526_0    bioconda
perl-encode               2.88                    pl526_1    bioconda
perl-encode-locale        1.05                    pl526_6    bioconda
perl-exporter             5.72                    pl526_1    bioconda
perl-exporter-tiny        1.002001                pl526_0    bioconda
perl-extutils-makemaker   7.36                    pl526_1    bioconda
perl-file-listing         6.04                    pl526_1    bioconda
perl-file-path            2.16                    pl526_0    bioconda
perl-file-temp            0.2304                  pl526_2    bioconda
perl-html-parser          3.72            pl526h04f5b5a_5    bioconda
perl-html-tagset          3.20                    pl526_3    bioconda
perl-html-tree            5.07                    pl526_1    bioconda
perl-http-cookies         6.04                    pl526_0    bioconda
perl-http-daemon          6.01                    pl526_1    bioconda
perl-http-date            6.02                    pl526_3    bioconda
perl-http-message         6.18                    pl526_0    bioconda
perl-http-negotiate       6.01                    pl526_3    bioconda
perl-io-compress          2.086           pl526h0a44026_0    bioconda
perl-io-html              1.001                   pl526_2    bioconda
perl-io-socket-ssl        2.066                   pl526_0    bioconda
perl-io-zlib              1.10                    pl526_2    bioconda
perl-json                 4.02                    pl526_0    bioconda
perl-json-xs              2.34            pl526h04f5b5a_3    bioconda
perl-libwww-perl          6.39                    pl526_0    bioconda
perl-list-moreutils       0.428                   pl526_1    bioconda
perl-list-moreutils-xs    0.428                   pl526_0    bioconda
perl-lwp-mediatypes       6.04                    pl526_0    bioconda
perl-lwp-protocol-https   6.07                    pl526_4    bioconda
perl-mime-base64          3.15                    pl526_1    bioconda
perl-mozilla-ca           20180117                pl526_1    bioconda
perl-net-http             6.19                    pl526_0    bioconda
perl-net-ssleay           1.88            pl526hb1dc21d_0    bioconda
perl-ntlm                 1.09                    pl526_4    bioconda
perl-parent               0.236                   pl526_1    bioconda
perl-pathtools            3.75            pl526h1de35cc_1    bioconda
perl-scalar-list-utils    1.50            pl526h1de35cc_0    bioconda
perl-socket               2.027                   pl526_1    bioconda
perl-storable             3.15            pl526h1de35cc_0    bioconda
perl-test-requiresinternet 0.05                    pl526_0    bioconda
perl-time-local           1.28                    pl526_1    bioconda
perl-try-tiny             0.30                    pl526_1    bioconda
perl-types-serialiser     1.0                     pl526_2    bioconda
perl-uri                  1.76                    pl526_0    bioconda
perl-www-robotrules       6.02                    pl526_3    bioconda
perl-xml-namespacesupport 1.12                    pl526_0    bioconda
perl-xml-parser           2.44            pl526h776b7f1_7    bioconda
perl-xml-sax              1.02                    pl526_0    bioconda
perl-xml-sax-base         1.09                    pl526_0    bioconda
perl-xml-sax-expat        0.51                    pl526_3    bioconda
perl-xml-simple           2.25                    pl526_1    bioconda
perl-xsloader             0.24                    pl526_0    bioconda
pip                       19.2.1                   py36_0    conda-forge
prodigal                  2.6.3                         1    bioconda
psutil                    5.4.3                    py36_0    conda-forge
pthread-stubs             0.4               h1de35cc_1001    conda-forge
pyani                     0.2.7            py36h24bf2e0_1    bioconda
pycparser                 2.19                     py36_1    conda-forge
pyopenssl                 19.0.0                   py36_0    conda-forge
pyparsing                 2.4.1.1                    py_0    conda-forge
pyqt                      5.9.2            py36h2a560b1_0    conda-forge
pyrsistent                0.15.3           py36h01d97ff_0    conda-forge
pysam                     0.15.2           py36h1719d13_3    bioconda
pysocks                   1.7.0                    py36_0    conda-forge
python                    3.6.7             h94afb7f_1005    conda-forge
python-dateutil           2.8.0                      py_0    conda-forge
python-levenshtein        0.12.0                   pypi_0    pypi
pytz                      2019.1                     py_0    conda-forge
pyyaml                    5.1.1            py36h01d97ff_0    conda-forge
qt                        5.9.7                h8cf7e54_3    conda-forge
ratelimiter               1.2.0                 py36_1000    conda-forge
readline                  8.0                  hcfe32e1_0    conda-forge
requests                  2.20.0                py36_1000    conda-forge
samtools                  1.9                 h7c4ea83_11    bioconda
scikit-learn              0.19.2           py36hebd9d1a_0  
scipy                     1.3.0            py36hab3da7d_0    conda-forge
seaborn                   0.9.0                      py_1    conda-forge
setuptools                41.0.1                   py36_0    conda-forge
sip                       4.19.8          py36h0a44026_1000    conda-forge
six                       1.11.0                py36_1001    conda-forge
snakemake-minimal         5.2.4                    py36_0    bioconda
sqlite                    3.29.0               hb7d70f7_0    conda-forge
statsmodels               0.9.0           py36h917ab60_1000    conda-forge
tabulate                  0.8.3                      py_0    conda-forge
tbb                       2019.7               h770b8ee_0    conda-forge
tk                        8.6.9             h2573ce8_1002    conda-forge
tornado                   6.0.3            py36h01d97ff_0    conda-forge
trimal                    1.4.1                h04f5b5a_3    bioconda
urllib3                   1.23                  py36_1001    conda-forge
wget                      1.20.1               hb1dc21d_0    conda-forge
wheel                     0.33.4                   py36_0    conda-forge
wrapt                     1.11.2           py36h01d97ff_0    conda-forge
xorg-kbproto              1.0.7             h1de35cc_1002    conda-forge
xorg-libice               1.0.10               h01d97ff_0    conda-forge
xorg-libsm                1.2.3             h01d97ff_1000    conda-forge
xorg-libx11               1.6.8                h01d97ff_0    conda-forge
xorg-libxau               1.0.9                h1de35cc_0    conda-forge
xorg-libxdmcp             1.1.3                h01d97ff_0    conda-forge
xorg-libxext              1.3.4                h01d97ff_0    conda-forge
xorg-libxrender           0.9.10            h01d97ff_1002    conda-forge
xorg-renderproto          0.11.1            h1de35cc_1002    conda-forge
xorg-xextproto            7.3.0             h1de35cc_1002    conda-forge
xorg-xproto               7.0.31            h1de35cc_1007    conda-forge
xz                        5.2.4             h1de35cc_1001    conda-forge
yaml                      0.1.7             h1de35cc_1001    conda-forge
zlib                      1.2.11            h01d97ff_1005    conda-forge
```

And `conda --version
conda 4.7.5`

Thanks Meren et al!

**Comment by ozcan** [comment]
Hi @scottdaniel,
You can find some suggestions about this issue here https://groups.google.com/d/msg/anvio/h-B_6w7kMco/gVk8o2lQBwAJ

**Comment by scottdaniel** [comment]
Hiya,

It looks like doing this fixed it:
`pip install https://github.com/scikit-learn/scikit-learn/archive/4035e60a6f0a0a2546bf0442ab603961c6a9cc4a.zip`
In the anvio conda environment

After it installed the developer version of scikit-learn, I ran the mini-tests and the `anvi-get-sequences-for-hmm-hits`:

```
anvi-get-sequences-for-hmm-hits --external-genomes external-genomes.txt \
>                                    --list-hmm-sources

HMM SOURCES COMMON TO ALL 12 GENOMES
===============================================
* Rinke_et_al [type: singlecopy] [num genes: 1975]
* Campbell_et_al [type: singlecopy] [num genes: 1731]
* Ribosomal_RNAs [type: Ribosomal_RNAs] [num genes: 244]
* BUSCO_83_Protista [type: singlecopy] [num genes: 1741]
```

**Comment by smsaladi** [comment]
Another fix in #1202
